documentation/writing_plans.md

```diff
@@ -391,6 +391,84 @@ plan loop(TargetSpec $targets) {
 
 If your plan accepts a single `TargetSpec` parameter you can call that parameter `targets` so that it can be specified with the `--targets` flag from the command line.
 
+### Creating Target objects
+
+Creating Target objects in a plan means they are part of the in-memory inventory - they can be
```

Comment:
style question. Should we use Target with a capital T without a code/monospace style?

Reply:
Yep, I caught that!